e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Customization', () => {
+    before(() => {
+        // # Login as sysadmin and visit customization system console page
+        cy.apiLogin('sysadmin');
+        cy.visit('/admin_console/site_config/customization');
+    });
+
+    it('SC20336 - Can change Custom Brand Image setting', () => {
+        // # Set Enable Custom Branding to true to be able to upload custom brand image
+        cy.get('[id="TeamSettings.EnableCustomBrandtrue"]').check();
```

Comment:
This selector might change, but I'm fine as is until we address - https://github.com/mattermost/mattermost-server/issues/13169